web-app/src/components/BookingModal/styled.js

```diff
@@ -64,3 +64,9 @@ export const FormContainer = styled.div`
     }
   }
 `;
+
+export const ButtonFloat = styled.div`
+  width: 25% !important;
```

Comment:
Are we using bootstrap? I'd try to stay away from setting custom percentage widths if so.